### PR TITLE
Docs typo and capitalization fixes

### PR DESCRIPTION
**Pull request by lzdanski**
Add code formatting and fix DAG capitalization.

### CODE REVIEW

File: docs/apache-airflow/howto/add-dag-tags.rst [hunk 1]
```diff
@@ -28,7 +28,7 @@ The filter is saved in a cookie and can be reset by the reset button.
 
 For example:
 
-In your Dag file, pass a list of tags you want to add to DAG object:
+In your DAG file, pass a list of tags you want to add to DAG object:
```
Comment (amoghrajesh) [inline docs/apache-airflow/howto/add-dag-tags.rst]: nit: `add to the DAG object`

**Review by amoghrajesh — APPROVED** [review]
Awesome. Congrats on your first PR!
One small nit.

File: docs/apache-airflow/howto/add-dag-tags.rst (hunk 1 above)
Comment (lzdanski) [reply docs/apache-airflow/howto/add-dag-tags.rst]: Thanks for catching! Fixed now.

**Review by jedcunningham — APPROVED** [review]

**Review by amoghrajesh — APPROVED** [review]
Looks good! +1

**Comment by jedcunningham** [comment]
Thanks @lzdanski! Congrats on your first commit 🎉

**Pull request merged**